addon-test-support/adapter.ts

```diff
@@ -0,0 +1,111 @@
+// FIXME: Remove
+import 'ember-source/types';
+import 'ember-source/types/preview';
```

Comment:
@chriskrycho I still need to look into the paths thing you mentioned, which would allow me to remove these